test/parser.test.js

```diff
@@ -1,9 +1,11 @@
+import { expect } from 'chai';
+import { Parser as UAPIParser, defaultConfig, errorsConfig } from '../src/Request/uapi-parser';
```

Comment:
See the comment about export.
After proposed change this will look like:
```javascript
import UAPIParser, { defaultConfig, errorsConfig } from '../src/Request/uapi-parser';
```